### ISSUE TITLE
Tests are not CultureInfo aware

### ISSUE THREAD

**Issue by ahorn42** [samcragg/sharpkml#18]
When running the `build.ps1` script seven test cases fail if you are using non-en locals (in my case German locals), because of a different symbol for decimal separation (`,` instead of `.`).

This not only results in 'wrong' format strings, but it also leads to wrong calculations due to "fuzzy parsing" results. `1.17` parsed as double results in `1.0` which clearly is wrong for parsing the number as double. (for example in the `ShouldTruncateInsignificantDigits` test case)

The easiest failing test case to start the search might be:
```csharp
[TestCase(1.17, ExpectedResult = "1.17")]
public string OutputShouldMatchInput(double value)
{
    return string.Format(KmlFormatter.Instance, "{0:G}", value);
}
```
which results in
```
Message:   String lengths are both 4. Strings differ at index 1.
  Expected: "1.17"
  But was:  "1,17"
  ------------^
```

It looks like the main issue lives in `KmlFormatter.cs`, but after a short view I could not exactly find the problem. Also the question is, what is the 'right' method. Localizing the test cases or using `CultureInfo.InvariantCulture` in the formatter. I am not sure what the intended behavior is.

**Comment by samcragg** [comment]
I've managed to reproduce and fix the issue for `OutputShouldMatchInput` (the KmlFormatter should not vary its output as the culture changes, so it is indeed a bug).

You mentioned there were seven tests failing, to make sure I've fixed them all can you list them here please so I can double check I've fixed them all?

Thanks

**Comment by ahorn42** [comment]
![sharpkml tests 2019-04-28 145348](https://user-images.githubusercontent.com/5106205/56864771-a25dad00-69c6-11e9-819d-1e359effe813.jpg)

it looks like I miscounted - it seems only to be 6 failing tests.

Thanks and best regards!

**Comment by samcragg** [comment]
Thanks for confirming that. This is now fixed in 5.0.1, which will be on NuGet shortly.

Thanks for reporting the issue and your help tracking down the cause 🙂
